packages/react-storage/src/components/StorageBrowser/views/LocationActionView/UploadControls.tsx

```diff
@@ -386,6 +388,9 @@ export const UploadControls = (): JSX.Element => {
       <Table
         data={tableData}
         columns={LOCATION_ACTION_VIEW_COLUMNS}
+        handleDroppedFiles={(files) => {
```

Comment:
Couldn't this just be:
```suggestion
        handleDroppedFiles={handleFileSelect}
```